client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -101,14 +101,19 @@ function NonBreakingSpace() {
   return <>{"\u00A0"}</>;
 }
 
-function labelFromString(version: string | boolean | null | undefined) {
+function labelFromString(
+  version: string | boolean | null | undefined,
```

Comment:
Is there a difference between the above and
```suggestion
  version?: string | boolean | null,
```
?